lib/fusion_auth/two_factor.ex

```diff
@@ -15,10 +15,49 @@ defmodule FusionAuth.TwoFactor do
   @type secret :: String.t()
   @type mobile_phone :: String.t()
   @type two_factor_id :: String.t()
-
+  @type email() :: String.t()
+  @type secret_base32_encoded() :: String.t()
+  @type method() :: String.t()
+  @type method_id() :: String.t()
+  @type application_id() :: String.t()
+  @type login_id() :: String.t()
+  @type state() :: map()
+  @type trust_challenge :: String.t()
```

Comment:
Just a minor consistency thing here. Some of these variables have brackets `()` and others do not. Doesn't matter to me one way or another, just looking for it to be consistent across the board